manim/mobject/shape_matchers.py

```diff
@@ -10,6 +10,23 @@
 
 
 class SurroundingRectangle(Rectangle):
+    """A rectangle surrounding a `mobject`
+
+    Examples
+    --------
+
+    .. manim:: SurroundingRectExample
+        :save_last_frame:
+
+        class SurroundingRectExample(Scene):
+            def construct(self):
+                title = Title("A Quote from Newton")
+                quote = Text("If I have seen further than others, \\nit is by standing upon the shoulders of giants.", color=BLUE).scale(0.75)
```

Comment:
This line is a bit too long, could you please break it up? Also, the escaped backslash actually confuses me, maybe using an r string would work as well instead?

```suggestion
                quote = Text(
                    r"If I have seen further than others, \n"
                    r"it is by standing upon the shoulders of giants.",
                    color=BLUE
                ).scale(0.75)
```